src/ape/_compat.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+# We can remove this once we stop supporting python3.7.
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
I don't think `typing_extensions` is in `setup.py`, although it is a dependency of one of our dependencies. I typically like to handle this in `install_requires` with `; < python_version<'3.8'` modifier

Reply:
Added! I am scared to pin a version lol... Because of what is happening in all the ethereum packages since 4.0.0 came out.

Reply:
Yeah, always an issue with their libraries, ugh